Collections/Aliasing Utilities/36400b92-ee54-4005-a413-eb57863edf8c - autoref.gvar

```diff
@@ -236,15 +243,16 @@
       }
     },
     "url": "roll",
-    "ts": "{\n    type: \\\"roll\\\";\n    dice: AnnotatedString;\n    name: string;\n    higher?: {int: string};\n    cantripScale?: boolean;\n    hidden?: boolean;\n}"
+    "ts": "{\n    type: \\\"roll\\\";\n    dice: AnnotatedString;\n    name: string;\n    higher?: {int: string};\n    cantripScale?: boolean;\n    hidden?: boolean;\n    displayName?: string;\n    fixedValue?: boolean;\n}"
   },
   "Text": {
     "desc": "Outputs a short amount of text in the resulting embed.",
     "opt": {
-      "text": "Either:\n\nAn AnnotatedString (the text to display).\n\nAn AbilityReference (see [AbilityReference](https://avrae.readthedocs.io/en/stable/automation_ref.html#abilityreference)). Displays the ability’s description in whole."
+      "text": "Either:\n\nAn AnnotatedString (the text to display).\n\nAn AbilityReference (see [AbilityReference](https://avrae.readthedocs.io/en/stable/automation_ref.html#abilityreference)). Displays the ability’s description in whole.",
+      "title": "*optional* - Allows you to set the name of the field. Defaults to “Effect”"
```

Comment:
fancy quotes should be \" instead

Reply:
I used fancy apostrophes for consistency since they were present in the GVAR prior to the PR. It's just within code blocks that the straight double-quotes are used.

Reply:
Oh hang on, I see which ones you were referring to now. Yeah, you're probably right